### PR TITLE
Parameter for the Databricks zone-id [skip ci]

### PR DESCRIPTION
**Pull request by NvTimLiu**
Parameter 'aws_zone' to switch to other zones when not having sufficient gpu resources in the requested zone.

```
{"code":"AWS_INSUFFICIENT_INSTANCE_CAPACITY_FAILURE","parameters":{"aws_api_error_code":"InsufficientInstanceCapacity","aws_error_message":"We currently do not have sufficient g4dn.2xlarge capacity in the Availability Zone you requested (us-west-2c)
```

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

File: jenkins/databricks/create.py [hunk 1]
```diff
@@ -36,21 +36,22 @@ def main():
...
       print(
-          'create.py -w <workspace> -t <token> -k <sshkey> -n <clustername> -i <idletime> -r <runtime> -o <workernodetype> -d <drivernodetype> -e <numworkers> -s <cloudprovider> -f <initscripts>')
+          'create.py -w <workspace> -t <token> -k <sshkey> -n <clustername> -i <idletime> -r <runtime> -o <workernodetype> -d <drivernodetype> -e <numworkers> -s <cloudprovider> -f <initscripts> -z <awszone>')
       sys.exit(2)
 
   for opt, arg in opts:
       if opt == '-h':
           print(
-              'create.py -w <workspace> -t <token> -k <sshkey> -n <clustername> -i <idletime> -r <runtime> -o <workernodetype> -d <drivernodetype> -e <numworkers> -s <cloudprovider>')
+              'create.py -w <workspace> -t <token> -k <sshkey> -n <clustername> -i <idletime> -r <runtime> -o <workernodetype> -d <drivernodetype> -e <numworkers> -s <cloudprovider>  -f <initscripts> -z <awszone>')
```
Comment (GaryShen2008) [inline jenkins/databricks/create.py:54]: You have 2 space before "-f".

File: jenkins/databricks/create.py [hunk 2]
```diff
@@ -95,7 +99,7 @@ def main():
       sys.exit(2)
 
   templ = ClusterUtils.generate_create_templ(sshkey, cluster_name, runtime, idletime,
-          num_workers, driver_type, worker_type, cloud_provider, init_scripts, printLoc=sys.stderr)
+          num_workers, driver_type, worker_type, cloud_provider, init_scripts, aws_zone,printLoc=sys.stderr)
```
Comment (GaryShen2008) [inline jenkins/databricks/create.py:102]: Need a space before "printLoc".

File: jenkins/databricks/create.py (hunk 1 above)
Comment (NvTimLiu) [reply jenkins/databricks/create.py]: Good catch, fixed

Comment (NvTimLiu) [reply jenkins/databricks/create.py]: fixed

File: jenkins/databricks/create.py (hunk 2 above)
Comment (NvTimLiu) [reply jenkins/databricks/create.py]: fixed

**Comment by NvTimLiu** [comment]
build

File: jenkins/databricks/create.py (hunk 1 above)
Comment (NvTimLiu) [reply jenkins/databricks/create.py]: Skip CI, local test and DB Jenkins pipelines PASS

**Comment by NvTimLiu** [comment]
Skip CI, local test and DB Jenkins pipelines PASS

**Review by jlowe — APPROVED** [review]

**Pull request merged**